web/src/beta/lib/reearth-ui/components/CodeInput/index.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import { action } from "@storybook/addon-actions";
+import { Meta, StoryObj } from "@storybook/react";
+
+import { CodeInput, CodeInputProps } from ".";
+
+const meta: Meta<CodeInputProps> = {
+  component: CodeInput,
+};
+
+export default meta;
+type Story = StoryObj<typeof CodeInput>;
+
+const TEST_JSON = {
+  name: "Eukarya",
+  application: "reearth",
+  component: "CodeInput",
+};
+
+export const Default: Story = {
+  render: () => (
+    <div style={{ height: "500px", width: "300px" }}>
+      <CodeInput
+        value={JSON.stringify(TEST_JSON)}
+        onChange={action("changed-value")}
```

Comment:
```suggestion
        onChange={action("onChange")}
```
I think it's better to keep the action name same as the event name.